components/alternate-accessions.js

```diff
@@ -0,0 +1,26 @@
+// node_modules
+import PropTypes from "prop-types";
+
+/**
+ * // Display the alternate accessions, normally below the header line in objects.
+ */
+export default function AlternateAccession({ alternateAccessions = [] }) {
+  if (alternateAccessions?.length > 0) {
+    return (
+      <h4 className="text-gray-500">
```

Comment:
Should the text be smaller, like on ENCODE? You can add the CSS class `text-sm` or `text-xs` if you think so.

Reply:
using text-sm